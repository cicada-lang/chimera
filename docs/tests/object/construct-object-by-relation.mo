clause Var(exp, name)
--------------- {
  exp = { "@type": "Exp", "@kind": "Var", name }
}

print find exp {
  Var(exp, "x")
}
