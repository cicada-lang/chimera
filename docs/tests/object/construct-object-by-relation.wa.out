[{ "@type": "Exp", "@kind": "Var", "name": "x" }]
